Samples/PrivateDns/ManagePrivateDns.cs

```diff
@@ -0,0 +1,197 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Management.Compute.Fluent;
+using Microsoft.Azure.Management.Compute.Fluent.Models;
+using Microsoft.Azure.Management.Fluent;
+using Microsoft.Azure.Management.Network.Fluent;
+using Microsoft.Azure.Management.ResourceManager.Fluent;
+using Microsoft.Azure.Management.ResourceManager.Fluent.Core;
+using Microsoft.Azure.Management.Samples.Common;
+using System;
+using System.Collections.Generic;
+
+namespace ManagePrivateDns
+{
+    public class Program
+    {
+        private const string CustomDomainName = "private.contoso.com";
+
+        /**
+         * Azure Private DNS sample for managing DNS zones.
+         *  - Creates a Private DNS Zone (private.contoso.com)
+         *  - Creates a Virtual Network
+         *  - Link a Virtual Network
+         *  - Creates test Virtual Machines
+         *  - Creates an additional DNS Record
+         *  - Test the private DNS zone
+         */
+        public static void RunSample(IAzure azure)
+        {
+            string rgName = SdkContext.RandomResourceName("MyARG", 24);
```

Comment:
Please keep the naming consistent with other areas. Thus, we can find the resource original or group.

Reply:
Done for the change